README.md

```diff
@@ -281,14 +286,37 @@ Please work with your enterprise network admin to ensure the appropriate firewal
 
 If you are not able to log in, please double check the settings that you have configured your OAuth application with, as well as the URLs that you have specified in your `bldr.env` file.
 
-You can also turn on debug logging (section below) and check to see that the authenticate endpoint is getting called at the Chef Habitat Builder API backend, and whether there is any additional information in the logs that may be helpful.
+### Unable to retrieve OAuth token
+
+You were able to sign in to the authentication provider, but unable authenticate with Chef Habitat's OAuth token.
+
+Open the `bldr.env` and verify that:
+* **APP_URL** ends with "/\"
+* **OAUTH_REDIRECT_URL** ends with "/\"
+* **OAUTH_CLIENT_ID** is complete and correct
+* **OAUTH_CLIENT_SECRET** is complete and correct
 
-The OAuth Token and API endpoints must be reachable from the Chef Habitat Builder on-prem install point.
+Apply changes to the to apply changes to the `bldr.env` by running the install script
 
-*Important*: If you change any settings in your `bldr.env` file, you will need to do the following steps after making the changes:
+```bash
+bash ./install.sh
+```
 
-1. Re-run the install script (`./install.sh`)
-2. Restart the services (`sudo systemctl restart hab-sup`)
+Restart the Chef Habitat services:
+
+```bash
+sudo systemctl restart hab-sup
+```
+
+### Self-signed cert files do not exist
+
+The latest version of the Chef Habitat Builder on-prem services looks certificates in the `/hab/cache/ssl` directory. Copy your self-signed certificates directory if they are missing. Follow the naming pattern `appname-cert.cert` or `appname-cert.pem`. Do not use `cert.pem`, which is reserved for the Chef Habitat system. Overwriting this file will cause Chef Habitat Builder to fail.
```

Comment:
"looks for certificates"?

Reply:
"Copy your self-signed certificated to the cache directory"?

Reply:
Is there any way we can get an example of this? what does "appname" need to correspond to?

Reply:
Sure! The naming pattern `appname-cert.cert` or `appname-cert.pem` is a largely arbitrary naming pattern. The important parts are 1) That it ends in `.cert` or `.pem` and 2) that it is NOT called `cert.pem`.  However, using a standard like `automate-cert.cert` or `company-cert.pem` lets us identify and troubleshoot quickly.